### ISSUE TITLE
failed with juicer's .hic format files

### ISSUE THREAD

**Issue by Linhua-Sun** [GangCaoLab/CoolBox#40]
Hi: 
I tried coolbox with .hic file.
```
test_hic = "MM_WT.FixedBin.hic"
dhic = DotHiC(test_hic, cmap="RdYlBu_r", depth_ratio=.5, resolution=10000)
frame = XAxis() + dhic
frame.plot("3:817500-2117500")
```

However, the output heatmap is empty. I am not sure what is the problem.

```
test_hic = "MM_WT.FixedBin.hic"
dhic = DotHiC(test_hic, cmap="RdYlBu_r", depth_ratio=.5, resolution=10000)
frame = XAxis() + dhic
frame.plot("3:817500-2117500")
test_hic = "MM_WT.FixedBin.hic"
dhic = DotHiC(test_hic, cmap="RdYlBu_r", depth_ratio=.5, resolution=10000)
frame = XAxis() + dhic
frame.plot("3:817500-2117500")
[WARNING:wrap.py:82 -   __fetch_straw_list()] Error occurred when reading the dothic file with straw:
[WARNING:wrap.py:83 -   __fetch_straw_list()] 'utf-8' codec can't decode byte 0x9a in position 1: invalid start byte
[WARNING:wrap.py:85 -   __fetch_straw_list()] Try to read unbalanced matrix.
[WARNING:wrap.py:90 -   __fetch_straw_list()] Failed.
[WARNING:wrap.py:91 -   __fetch_straw_list()] 'utf-8' codec can't decode byte 0x9a in position 1: invalid start byte
[WARNING:wrap.py:92 -   __fetch_straw_list()] Unbalanced matrix is readed.
[WARNING:plot.py:254 -     matrix_val_range()] zero-size array to reduction operation minimum which has no identity
```
Thanks.

**Comment by zhqu1148980644** [comment]
Hi. If you are using the latest version of CoolBox on GitHub? This may be caused by older/newer veriosn of `.hic` file,  we met this kind of error before and @Nanguage has changed the code at some point.

**Comment by Linhua-Sun** [comment]
Yes, I am using  the latest version of CoolBox (0.3.7) on GitHub. 
And I used Juicer Tools Version 1.9.9 to generated .hic files from validpairs of hic-pro.

**Comment by zhqu1148980644** [comment]
How about converting .hic to .cool file with https://github.com/4dn-dcic/hic2cool， this may be the fatest way to solve your problem.

**Comment by Linhua-Sun** [comment]
Thanks! I have tested it with hic2cool. It works！While, using .hic will be easier to generated some plots like https://gangcaolab.github.io/CoolBox/_gallery/reso_normalize.html.

**Comment by zhqu1148980644** [comment]
I think Cool track has all functionalities that DotHic has.

**Comment by Linhua-Sun** [comment]
OK, thanks for you answer. I  will test it will cool files.

**Comment by Nanguage** [comment]
I think I have fixed this bug in the new version(0.3.8).
![image](https://user-images.githubusercontent.com/14010939/120145635-6dac2780-c216-11eb-8b73-19b29dcc64fa.png)

Update from GitHub source code to use this version: https://gangcaolab.github.io/CoolBox/installation.html

**Comment by Linhua-Sun** [comment]
Thank you so much! That's cool!
